### DISCUSSION TITLE
Sound not working - no ideas left.

### DISCUSSION THREAD

**Discussion by selmaohneh** [MiczFlor/RPi-Jukebox-RFID#1394] [Q&A] [answered]
Hi!

Just made a fresh install on my Raspi 4 with a HifiBerry DAC HAT using the one-line-install-script.
The web app shows up and even Spotify is working. 
The sound works when I test it via `speaker-test`. But somehow Phoniebox does not play a sound at all, not even the startup sound.

What I have done after that:

* I followed the hifiberry documentation and did the following steps: https://www.hifiberry.com/docs/software/configuring-linux-3-18-x/

`aplay -l` now gives the following output: 

```
**** List of PLAYBACK Hardware Devices ****
card 0: sndrpihifiberry [snd_rpi_hifiberry_dac], device 0: HifiBerry DAC HiFi pcm5102a-hifi-0 [HifiBerry DAC HiFi pcm5102a-hifi-0]
  Subdevices: 1/1
  Subdevice #0: subdevice 
```

* I did the following instructions: https://github.com/MiczFlor/RPi-Jukebox-RFID/wiki/MANUAL#settingsaudio_iface_name . 
Since `amixer scontrols` returns the `Simple mixer control 'Master',0
Simple mixer control 'Capture',0`, I set `Master` as my iFace name inside `settings/Audio_iFace_Name`, `settings.conf` and `/etc/mpd.conf`. This is how the section in the latter one looks right now:

```
audio_output {
        type            "alsa"
        name            "My ALSA Device"
#       device          "hw:0,0"        # optional
#       mixer_type      "hardware"      # optional
#       mixer_device    "default"       # optional
        mixer_control   "Master"                # optional
#       mixer_index     "0"             # optional
```

* I read the following: https://github.com/MiczFlor/RPi-Jukebox-RFID/wiki/HiFiBerry-Soundcard-Details

The only additional info here was the GPIO issue. The pins 19 and 21 are currently not in use, i.e. `enabled: False`. So I guess this is not in issue. Nevertheless I tried to change them too. Still no success.

Anyone willing to help? What am I doing wrong / missing?

Thanks in advance!

**Comment by selmaohneh** [comment, marked as answer]
It was the hardware! Swapping to a new hifiberry did the trick for me.

**Reply by MiczFlor** [reply]
The one in a million chance. You are a lucky person :)
